### DISCUSSION TITLE
The new version function is too difficult to use

### DISCUSSION THREAD

**Discussion by kajweb** [mockoon/mockoon#642] [Help] [answered]
Bug-laden version for users of migrating systems
Import function is not available
The opening function of the new version is sometimes abnormal

**Comment by 255kb** [comment]
Could you please elaborate on what issue do you encountered? Without more details we won't be able to help you.
Thank you.

**Comment by kajweb** [comment]
First of all, the configuration of the old version can be exported normally, but the exported file cannot be used in this version.

For Chinese file names, such as "测试.json", it will be automatically encoded as "%E6%B5%8B%E8%AF%95.json" when saving. (The correct file name is recorded in "storage/settings.json")

In the new version, open environment does not support opening folders. But in most cases all my environments should be in the same folder.
For a single profile, sometimes when I open an environment and close it, it's hard to find the environment I want in a ton of files. So I'm not very supportive of opening a single file as json

**Comment by 255kb** [comment, marked as answer]
"Open environment" never supported folders. If you want it to support folders (i.e. opening all the files in a folder), please open a new issue. 
Also, the import/export was never designed to be a replacement for opening multiple environments at once. This feature will be removed in the next version as it offers no advantage over directly opening environment files, only more confusion. 

Concerning the file names in chinese, we never tested this use case. I suggest you open a new bug issue so we can investigate this.
